### DISCUSSION TITLE
Can I use useBetween in react native?

### DISCUSSION THREAD

**Discussion by MohammedTahirr** [betula/use-between#41] [Q&A] [answered]
Can I use this library in react native?

**Comment by betula** [comment, marked as answer]
Hello @MohammedTahirr, I have great news!
Yes, useBetween is **fully compatible with react-native**.
You can use it with pleasure!

I tested on several versions and everything works fine!, an example can be found here:
https://github.com/betula/use-between-react-native-ok

Thanks for using it!
And enjoy your Code!

**Reply by MohammedTahirr** [reply]
Thanks alot mate
